<launch>
    <arg name="camera_image_topic" default="camera_node/image/compressed"/>
    <arg name="camera_info_topic" default="camera_node/camera_info"/>
    <arg name="apriltags_topic" default="/poses_acquisition/poses"/>
    <arg name="processors_num" default="4"/>
    <arg name="marker_size" default="0.065"/>

    <node name="apriltag_processor_node" type="apriltag_processor_node"
          pkg="processing_node" output="screen" required="true">
        <param name="config_file" value="$(find processing_node)/config/config.yml"/>
        <param name="camera_image_topic" value="$(arg camera_image_topic)"/>
        <param name="camera_info_topic" value="$(arg camera_info_topic)"/>
        <param name="apriltags_topic" value="$(arg apriltags_topic)"/>
        <param name="processors_num" value="$(arg processors_num)"/>
        <param name="marker_size" value="$(arg marker_size)"/>
    </node>
</launch>